sdk/communication/Azure.Communication.Identity/README.md

```diff
@@ -0,0 +1,137 @@
+# Azure Communication Identity client library for .NET
+
+> Server Version:
+> Identity client: 2021-03-07
+
+Azure Communication Identity is managing tokens for Azure Communication Services.
+
+[Source code][source] <!--| [Package (NuGet)][package]--> | [Product documentation][product_docs] | [Samples][source_samples]
```

Comment:
I had to comment the package link out for now as the package is not published on NuGet yet.